app/auth/login/index.tsx

```diff
@@ -0,0 +1,94 @@
+import { LOGIN_MUTATION, ORG_LOGIN_MUTATION } from '@/graphql/mutations/login.mutation';
+import { UserContext } from '@/hook/useAuth';
+import { useMutation } from '@apollo/client';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import { useRouter } from 'expo-router';
+import { useContext, useState } from 'react';
+import { ToastAndroid, Alert } from 'react-native';
+
+export default function SignInOrganization() {
+  const router = useRouter();
+  const [orgLoginSuccess, setOrgLoginSuccess] = useState(false);
+  const [orgLoginMutation] = useMutation(ORG_LOGIN_MUTATION);
+  const [loading, setLoading] = useState(false);
+  const { login } = useContext(UserContext) || {};
+  const [LoginUser] = useMutation(LOGIN_MUTATION);
+
+  const onOrgSubmit = async (values: any) => {
+    try {
+      await orgLoginMutation({
+        variables: {
+          orgInput: {
+            name: values.organization,
+          },
+        },
+        onCompleted(LoginUser) {
+          setOrgLoginSuccess(true);
+        },
+        onError(err: any) {
+          ToastAndroid.show(`${err}`, ToastAndroid.LONG);
+        },
+      });
+    } catch (err) {
+      console.log('Mutation error:', err);
+    }
+  };
+
+  const onSubmit = async (userInput: any) => {
+    try {
+      const orgToken = await AsyncStorage.getItem('orgToken');
+      userInput.orgToken = orgToken;
+      setLoading(true);
+      await LoginUser({
+        variables: {
+          loginInput: userInput,
+        },
+        onCompleted: async (data) => {
+          if (data.addMemberToCohort) {
+            ToastAndroid.show(`${data.addMemberToCohort}`, ToastAndroid.LONG);
+          }
+          if (login && data.loginUser) {
+            login(data.loginUser);
+            await client.resetStore();
+            Alert.alert('Welcome');
+          }
+          if (data.loginUser) {
+            const role = data.loginUser.user.role;
+            if (role === 'admin' || role === 'coordinator') {
+              router.push('/trainee');
+            } else {
+              Alert.alert('The app is for the trainee only');
+            }
+          } else {
+            router.push('/dashboard');
+          }
+        },
+        onError: (err) => {
+          if (err.networkError) {
+            ToastAndroid.show('There was a problem contacting the server', ToastAndroid.LONG);
+          } else if (err.message.toLowerCase() !== 'invalid credential') {
+            const translateError = 'Please wait to be added to a program or cohort';
+            Alert.alert(translateError);
+          } else {
+            console.log('password', {
+              type: 'custom',
+              message: 'Invalid credentials',
+            });
+          }
+        },
+      });
+    } catch (error: any) {
+      console.log('password', {
+        type: 'custom',
+        message: 'Invalid credentials',
+      });
+    } finally {
+      setLoading(false);
```

Comment:
- Can we have a method/class for error handler?
- Can you remove all logs

Reply:
### Done.